.devcontainer/Dockerfile

```diff
@@ -0,0 +1,91 @@
+FROM mcr.microsoft.com/devcontainers/base:jammy
+
+RUN export DEBIAN_FRONTEND=noninteractive && \
+    apt-get update && \
+    apt-get install --no-install-recommends -y \
+        acl \
+        bash-completion \
+        build-essential \
+        curl \
+        enchant-2 \
+        gawk \
+        gcc \
+        genisoimage \
+        git \
+        isomd5sum \
+        libassuan-dev \
+        libbtrfs-dev \
+        libdevmapper-dev \
+        libgdbm-dev \
+        libgpgme-dev \
+        libncursesw5-dev libxml2-dev libxmlsec1-dev libffi-dev liblzma-dev \
+        libssl-dev libbz2-dev libreadline-dev libsqlite3-dev \
+        make \
+        npm \
+        pkg-config \
+        plantuml \
+        python3.10-gdbm \
+        shellcheck \
+        tk-dev \
+        tmux \
+        vim \
+        xz-utils \
+        zlib1g-dev \
+        && \
+    apt-get clean
+
+USER vscode
+
+# Install helm
+ARG HELM_VERSION=3.14.4
+
+RUN curl --fail -L -o /tmp/helm.tar.gz https://get.helm.sh/helm-v${HELM_VERSION}-linux-amd64.tar.gz && \
+    sudo tar -zxvf /tmp/helm.tar.gz -C /usr/local/bin/ linux-amd64/helm --strip-components 1 && \
+    rm -rf /tmp/helm.tar.gz
+
+    sudo tar -xzf /tmp/crane.tar.gz -C /usr/local/bin/ crane && \
+    rm -rf /tmp/crane.tar.gz
+
+# Install go
+ARG GO_VERSION=1.20.14
+
+RUN curl --fail -L -o /tmp/go.tar.gz https://go.dev/dl/go${GO_VERSION}.linux-amd64.tar.gz && \
+    sudo tar -zxvf /tmp/go.tar.gz -C /usr/local/bin/ go/bin/go --strip-components 2 && \
+    rm -rf /tmp/go.tar.gz
```

Comment:
maybe use 1.19 like for operator 
codegen needs to be tested with this

Reply:
as discussed : 
- 1.19 -->127
- 1.20 --> 128